packages/thirdweb/src/extensions/erc721/read/getNFTs.ts

```diff
@@ -132,8 +132,14 @@
     ),
   );
   if (result.length < expectedResultLength) {
-    // fresh contracts might be delayed in indexing, so we fallback to RPC
-    return getNFTsFromRPC(options);
+    try {
+      // fresh contracts might be delayed in indexing, so we fallback to RPC
+      const rpcResult = await getNFTsFromRPC(options);
+      return rpcResult;
```

Comment:
just return it directly? why the re-assignment?

Reply:
can't return it - we won't be able to catch the error, have to await it

Reply:
updated